package.json

```diff
@@ -12,14 +12,14 @@
   },
   "scripts": {
     "format": "prettier '**/{*.{js?(on),ts?(x),graphql,md,scss},.*.js?(on)}' --write --list-different --config prettier.config.js",
-    "format:check": "yarn run format --write=false --check --list-different=false --loglevel=warn",
+    "format:check": "pnpm run format --write=false --check --list-different=false --loglevel=warn",
     "_lint:js": "DOCSITE_LIST=\"$(./dev/docsite.sh -config doc/docsite.json ls)\" NODE_OPTIONS=\"--max_old_space_size=16192\" eslint",
-    "lint:js:changed": "yarn _lint:js $(git diff --diff-filter=d --name-only origin/main... | grep -E '\\.[tj]sx?$' | xargs)",
-    "lint:js:root": "yarn run _lint:js --quiet '*.[tj]s?(x)'",
-    "lint:js:all": "DOCSITE_LIST=\"$(./dev/docsite.sh -config doc/docsite.json ls)\" dev/foreach-ts-project.sh yarn run lint:js --quiet && yarn lint:js:root",
+    "lint:js:changed": "pnpm run _lint:js $(git diff --diff-filter=d --name-only origin/main... | grep -E '\\.[tj]sx?$' | xargs)",
+    "lint:js:root": "pnpm run _lint:js --quiet '*.[tj]s?(x)'",
+    "lint:js:all": "DOCSITE_LIST=\"$(./dev/docsite.sh -config doc/docsite.json ls)\" dev/foreach-ts-project.sh pnpm lint:js --quiet && pnpm lint:js:root",
     "_lint:css": "stylelint --quiet",
```

Comment:
Here's the other alternative to `pnpm [run]`... For package bin entries you can just run it directly without even putting `pnpm` in the command, I guess this also works for `yarn` since it was already used here.

Should we do this where possible to avoid putting the word `yarn` or `pnpm` etc into the source? Then this diff would have been smaller...

Reply:
Why don't we do the minimal syntax possible everywhere and see if that works out.

Reply:
Ok I've updated this to the minimum which is
* `pnpm [script]` to run things in the package.json `scripts`
* `abcd` to run the npm `abcd` bin entry such as `eslint`

So never `pnpm run` since the `run` seems optional, and never `pnpm [npm-pkg]` since the npm `bin` directory is on the `PATH`